src/graphql/schema.graphql

```diff
@@ -24,7 +30,43 @@ type Mutation {
 
     "Whether or not the contributor wants to receive email updates."
     subscribe: Boolean!
-  ): DiscountCode!
+  ): DiscountCode! @deprecated(reason: "use createContributor instead")
+
+  createContributor(input: CreateContributorInput): Contributor
+
+  addTagsToShopifyCustomer(id: String!, tags: [String!]!): Boolean!
```

Comment:
This shouldn't be public; adding arbitrary tags via a public API is just begging to be abused.